node-operators/appchain-node/rpc-systemd.md

```diff
@@ -0,0 +1,265 @@
+---
+
+# Run an Appchain Node Using Systemd
+
+## Introduction {: #introduction }
+
+Running a Tanssi Appchain node allows you to connect to and interact with the network using your infrastructure via either HTTP or WebSocket protocols. 
+
+Nodes store block data and network state. However, developers can run different kinds of nodes:
+ 
+ - **Full Archive Node** - a node storing the entire block data and network state at all block heights. Such nodes are helpful when querying historical data from old blocks. However, a full archive node takes up a lot of space
+ 
+  - **Full Pruned Node** - a node storing block data and network state up to some specific number of blocks before the current block height. Such nodes are helpful when querying recent data or submitting transactions through your infrastructure. They require much less space than an archival node but don't store the full network state
+
+In this guide, you'll learn how to spin up a Tanssi Appchain node using a binary executable file and managing the service with [Systemd](https://systemd.io/){target=\_blank} on Linux systems.
+
+!!! note
+    It is not possible to run an RPC node for Snap Appchains as they run on a private network, and their nodes are, therefore, unreachable for syncing.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To get started, you'll need access to a computer running an Ubuntu Linux OS and root privileges. You will also need:
+
+- **Node binary file** - the instructions in this guide execute the [latest](https://github.com/moondance-labs/tanssi/releases/latest){target=\_blank} official stable node release. If you want to build and run your own file, make sure to follow the instructions for [building your Appchain node](/builders/build/customize/prerequisites){target=\_blank}
+
+- **Appchain specifications file** - the Appchain specification file is needed to run the node. You can download it from the dashboard in the [dApp](https://apps.tanssi.network/){target=\_blank} by clicking the `Appchain Data` link
+
+    ![Getting the chain specs](/images/node-operators/appchain-node/rpc-systemd/rpc-systemd-1.webp)
+
+- **Relay chain specifications file** - the relay chain specification file can be downloaded from this [public GitHub repository](https://github.com/papermoonio/external-files/blob/main/Moonbeam/Moonbase-Alpha/){target=\_blank}
+
+## Download the Latest Release {: #download-latest-release }
+
+Every new release includes two different node binaries, one for EVM-compatible Appchains and another one for Substrate Appchains. To get started, run the following command to get the latest release binary that matches your Appchain type and make it executable:
+
+=== "EVM-Compatible Appchain"
+
+    ```bash
+    wget https://github.com/moondance-labs/tanssi/releases/latest/download/container-chain-template-frontier-node && \
+    chmod +x ./container-chain-template-frontier-node
```

Comment:
I think it is important that we state somewhere that these steps must be adapted to your specific infra-running configuration or something like that.

For example, in Moonbeam we do it via a `moonbeam.service` and then we do `chmod` but a bit differently (for the service only, not general. 

https://docs.moonbeam.network/node-operators/networks/run-a-node/systemd/#setup-the-service

Reply:
Added a few lines in the intro, inviting the reader to adapt the guide